packages/core/src/components/Modal/Modal.module.scss

```diff
@@ -27,7 +27,6 @@
 }
 
 .dialog {
-  z-index: 2;
```

Comment:
Not needed, and makes additional z-index issues inside the modal which increases complexity for no reason